VK/VK/Model/News/Attachments.swift

```diff
@@ -4,7 +4,7 @@
 import Foundation
 
 /// Вложения новости
-struct ItemAttachment: Decodable {
+struct Attachment: Decodable {
```

Comment:
```suggestion
struct Attachment: Decodable {
```
Название файла AttachmentS